operatorcert/utils.py

```diff
@@ -21,3 +21,18 @@ def find_file(
         if new_path.exists() and new_path.is_file():
             return new_path
     return None
+
+
+def str_color(color, data):
```

Comment:
There is already a colored logger so I guess we don't have to re-implement a wheel again: https://pypi.org/project/coloredlogs/

Although I am not really sure if this is something that we need to have in our pipelines. What's an additional value?

Reply:
The only value is color :slightly_smiling_face: 
I can remove it, I only wanted to make the output that is important for users to see more visible.

Reply:
I removed the colorful logs. Maybe eventually, one day we'll add them for all the scripts.